nodexml/README.md

```diff
@@ -0,0 +1,79 @@
+Node XML Schema
+===============
+
+
+Work-In-Progress
+----------------
+
+This readme, the schema, and the example document/s are all still a
+work-in-progress. Major changes are expected to take place.
+
+
+About
+-----
+
+This schema defines what the UNT Libraries considers to be a valid
+nodexml instance. A nodexml instance is an XML representation of a
+storage node. Specifically, the nodes represented are the storage
+nodes for bags, which follow the 
+[BagIt](http://www.digitalpreservation.gov/documents/bagitspec.pdf)
+standard.
+
+
+Requirements
+------------
+
+
+### Namespace ###
+
+The namespace for all elements of a valid nodexml instance must be set to
+"http://digital2.library.unt.edu/coda/nodexml/".
+
+
+### node ###
+
+This is the root element of a nodexml instance. This root element must contain
+a single instance of each of the following child elements:
+
+* name
+* url
+* path
+* capacity
+* size
+* lastChecked
+
+
+### name ###
+
+The name of the storage node.
+
+
+### url ###
+
+The full URL of the storage node.
+
+
+### path ###
+
+The 
```

Comment:
Yeah, the current description is quite unhelpful. Didn't realize I hadn't finished that.